src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -117,10 +117,14 @@ export default [
         shouldShow: (type, reportAction, isArchivedRoom, betas, anchor, isChronosReport, reportID) =>
             type === CONTEXT_MENU_TYPES.REPORT_ACTION && reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT && !ReportUtils.isThreadFirstChat(reportAction, reportID),
         onPress: (closePopover, {reportAction, reportID}) => {
-            Report.navigateToAndOpenChildReport(lodashGet(reportAction, 'childReportID', '0'), reportAction, reportID);
             if (closePopover) {
-                hideContextMenu(true, ReportActionComposeFocusManager.focus);
```

Comment:
Removed `ReportActionComposeFocusManager.focus` because it doesn't do anything. We can't focus on the composer while navigating. Composer also already has its autofocus (but it has an [issue](https://github.com/Expensify/App/issues/19558) in native).
https://github.com/Expensify/App/blob/820fa93ad1e799c12d8b639b98018b5350ea17af/src/pages/home/report/ReportActionCompose.js#L1072-L1073

Reply:
Added back